src/odemis/util/spot.py

```diff
@@ -255,17 +256,19 @@ def MaximaFind(image, qty, len_object=18):
 
     # dilation
     structure = _CreateSEDisk(len_object // 2)
-    dilated = cv2.dilate(filtered, structure, iterations=1)
+    dilated = cv2.dilate(filtered, structure)
 
     # find local maxima
-    binary = (filtered == dilated)
+    binary = (filtered == dilated).astype(numpy.uint8)
```

Comment:
Could you explain what is this needed? Normally it returns an array of booleans. Maybe that is not compatible with cv2? (I guess that booleans are stored as uint8 anyway, so that's why it'd work anyway).

Reply:
Yes; I think this has to do with cv2. Note that I didn't added the .astype() call; I just moved it up by a few lines. As you say it might indeed just work without but I think it is better to explicitly cast the array to what is expected by cv2.

Reply:
Yes, this is because of cv2. It doesn't accept it without the .astype() call